### DISCUSSION TITLE
Welcome to Pixelfed App Discussions!

### DISCUSSION THREAD

**Discussion by dansup** [pixelfed/pixelfed-rn#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by insxa** [comment]
Hello where can i download beta version?

**Comment by Wolle980** [comment]
I'm using this App as 1.0.2.78 but I'm missing a German Translation of menu and so. Can I help for it? 🤫 Is there an interface where I could supply German terms?
Kind regards

**Reply by Kwasow** [reply]
This is something we're working on. We'll let you know if we need any support from you!
